data/find-questions.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * To use this:
+ * - Modify `predicateCallback` to look for questions that
+ *   match your requirements
+ * - run: node find-questions.js
```

Comment:
One idea.  If you want to use TypeScript and ES module imports in this file, you could:
  * Rename it to `.ts`
  * Add `#!/usr/bin/env -S node -r @swc-node/register` as the first line in this file
  * Run `chmod +x dev/find-questions.js`

Then we'd be able to use types here and run it by simply doing `./dev/find-questions.js`.  You can see an example of this in `utils/extract-strings.ts`

Totally optional though!

Reply:
Done, thanks

Reply:
TIL: `#!/usr/bin/env -S node -r @swc-node/register`

That's a neat trick!